tests/integration_tests/with_db/deployment/test_deploy_cli.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+from click.testing import CliRunner
+import pyexasol
+import exasol.bucketfs as bfs
+from exasol.pytest_backend import BACKEND_ONPREM
+from exasol.python_extension_common.cli.std_options import StdParams, get_cli_arg
+from exasol.python_extension_common.deployment.language_container_validator import temp_schema
+
+from exasol_transformers_extension.deploy import (
+    deploy_command, DEPLOY_SLC_ARG, BUCKETFS_CONN_NAME_ARG, get_bool_opt_name)
+from exasol_transformers_extension.deployment.language_container import export_slc
+from tests.integration_tests.with_db.test_upload_model import run_model_upload_test
+
+PATH_IN_BUCKET = 'te_end2end'
+BUCKETFS_CONN_NAME = 'TE_E2E_BFS_CONN'
+LANGUAGE_ALIAS = 'TE_E2E_LANG_ALIAS'
+
+
+@pytest.mark.skip('Need to sort out the model upload test first')
+def test_deploy_cli(pyexasol_connection,
+                    backend_aware_database_params,
+                    backend_aware_bucketfs_params,
+                    bucketfs_cli_args,
+                    cli_args):
+
+    with temp_schema(pyexasol_connection) as schema_name:
```

Comment:
Would it make sense to create a fixture `temp_schema`?

Reply:
I don't think so. Why would we need a fixture?

Reply:
We don't need a fixture, but it could be beneficial IMHO
* Using the fixture `temp_schema` can replace using fixture `pyexasol_connection` 
* and also replaces the `with` statement in the test.
* reduces the indent level in the test case

Reply:
We can consider adding a test-level fixture `temp-schema` or a pyexasol connection opened with a temporary schema into the pytest plugins. For now, I think it's alright as it is.

Reply:
:+1: